.github/workflows/testing-and-deployment.yml

```diff
@@ -135,8 +135,6 @@ jobs:
       # see discussion at https://github.com/pyvista/pyvista/issues/2867
       matrix:
         include:
-          - python-version: '3.7'
```

Comment:
Since we have recently dropped support for these, it is a little concerning that we have to drop testing support for this release.  Is there some change that breaks either 3.7 support or vtk8 support?

Reply:
Failing due to:
```
Image cache is only valid for VTK9+
```

I'll have to go back to `pytest-pyvista==0.1.6`.

Reverting...

Reply:
The problem is on removing the `skip_image_cache_vtk8` for 0.1.6